### PR TITLE
Setup deployment

### PR DESCRIPTION
**Pull request by asvetlov**

### CODE REVIEW

File: CHANGES/622.feature.3.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+Started shipping platform-specific wheels with the ``musl`` tag targeting typical Alpine Linux runtimes.Started shipping platform-specific arm64 wheels for Apple Silicon.
```
Comment (webknjaz) [inline CHANGES/622.feature.3.rst:1]:
```suggestion
Started shipping platform-specific arm64 wheels for Apple Silicon.
```

File: CHANGES/622.feature.1.rst [hunk 2]
```diff
@@ -0,0 +1 @@
+Added support Python 3.10
```
Comment (webknjaz) [inline CHANGES/622.feature.1.rst:1]:
```suggestion
Added support for Python 3.10.
```

File: CHANGES/README.rst [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+a change log for the respective version. It is important to
+understand that including unnecessary low-level implementation
+related details generates noise that is not particularly useful
+to the end-users most of the time. And so such details should be
+recorded in the Git history rather than a changelog.
+
+Alright! So how to add a news fragment?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+``multidict`` uses `towncrier <https://pypi.org/project/towncrier/>`_
```
Comment (webknjaz) [inline CHANGES/README.rst:24]:
```suggestion
``yarl`` uses `towncrier <https://pypi.org/project/towncrier/>`_
```

**Comment by asvetlov** [comment]
Thnks, @webknjaz 
Your suggestions are applied

**Pull request merged**